src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -164,6 +164,25 @@ export default function ConfirmedAppointmentDetailsPage() {
     facility: facilityData[locationId],
   });
 
+  let alertBox = null;
+  if (isPastAppointment) {
```

Comment:
I think we'd want to have the canceled alert have first priority here. It's odd to say that an appointment occurred in the past if it was canceled and never occurred.

Reply:
Done.